cfgov/core/tests/test_forms.py

```diff
@@ -0,0 +1,47 @@
+from django.test import TestCase, RequestFactory
+from django.conf import settings
+
+from core.utils import sign_url
+from core.forms import ExternalURLForm
+
+
+class TestExternalURLForm(TestCase):
+    def setUp(self):
+        # insulate the test from future whitelist changes
+        self.preserved_whitelist = settings.EXTERNAL_URL_WHITELIST
+        settings.EXTERNAL_URL_WHITELIST = (r'^https:\/\/facebook\.com\/cfpb$',)
```

Comment:
Could also use [`@modify_settings`](https://docs.djangoproject.com/en/1.8/topics/testing/tools/#django.test.override_settings) decorator on the test class here, avoiding the need for separate `setUp`/`tearDown`.